helios-testing/src/test/java/com/spotify/helios/testing/DefaultDeployerTest.java

```diff
@@ -42,10 +41,9 @@
 import static java.util.Collections.emptyMap;
 import static java.util.concurrent.TimeUnit.MINUTES;
 import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.fail;
+import static org.mockito.Mockito.mock;
 import static org.mockito.Mockito.when;
 
-@RunWith(MockitoJUnitRunner.class)
```

Comment:
it takes the fields annotated with `@Mock` and sets their value to `Mockito.mock(TheClass.class)` at setup-time. I replaced it here because it felt like less code to just create the one mock myself in line 70 (and moreso because then I couldn't initialize the DefaultDeployer in a single line and I didn't want to add a `@Before` method :grinning: )